docs/source/robots/arms/index.md

```diff
@@ -31,4 +31,17 @@ Robot UID: `widowx250s`
 
 Agent Class Code: [https://github.com/haosulab/ManiSkill/blob/main/mani_skill/agents/robots/widowx/widowx.py](https://github.com/haosulab/ManiSkill/blob/main/mani_skill/agents/robots/widowx/widowx.py)
 
-Robot Description: [https://github.com/haosulab/ManiSkill-WidowX250S](https://github.com/haosulab/ManiSkill-WidowX250S)
\ No newline at end of file
+Robot Description: [https://github.com/haosulab/ManiSkill-WidowX250S](https://github.com/haosulab/ManiSkill-WidowX250S)
+
+## Widow X AI
```

Comment:
thanks for documenting!

Meanwhile need to put up an issue to auto-generate the robot docs like the task docs in the future to make things easier to maintain.

Reply:
:memo:

I feel like github is probably working towards some kind of github action agent that does exactly this